elide-standalone/src/main/java/com/yahoo/elide/standalone/Util.java

```diff
@@ -89,4 +133,39 @@ public static EntityManagerFactory getEntityManagerFactory(String modelPackageNa
                 .map(Class::getName)
                 .collect(Collectors.toList());
     }
+
+    /**
+     * Find classes with a particular annotation from dynamic compiler.
+     * @param compiler An instance of ElideDynamicEntityCompiler.
+     * @param annotationClass Annotation to search for.
+     * @return Set of Classes matching the annotation.
+     * @throws ClassNotFoundException
+     */
+    @SuppressWarnings({ "unchecked", "rawtypes" })
+    public static List<String> findAnnotatedClasses(ElideDynamicEntityCompiler compiler, Class annotationClass)
+            throws ClassNotFoundException {
+        List<String> annotatedClass = new ArrayList<String>();
+        List<String> dynamicClasses = ElideDynamicEntityCompiler.classNames;
+        for (String dynamicClass : dynamicClasses) {
+            Class<?> classz = compiler.getClassLoader().loadClass(dynamicClass);
+            if (classz.getAnnotation(annotationClass) != null) {
+                annotatedClass.add(classz.getName());
+            }
+        }
+        return annotatedClass;
+    }
+
+    @SuppressWarnings({ "unchecked", "rawtypes" })
+    public static Set<Class> populateBindClasses(ElideDynamicEntityCompiler compiler,
+            Class annotationClass) throws ClassNotFoundException {
+        Set<Class> bindClasses = new HashSet<>();
+        List<String> dynamicClasses = ElideDynamicEntityCompiler.classNames;
+        for (String dynamicClass : dynamicClasses) {
+            Class<?> bindClass = compiler.getClassLoader().loadClass(dynamicClass).getClass();
```

Comment:
Do we need to use getClass() here, wont doing just loadClass work?

Reply:
if only loadClass works, there is common code between populateBindClasses and findAnnotatedClasses. Maybe we call populateBindClasses in findAnnotatedClasses and then traverse the list to get the names?